api/swagger.yml

```
@@ -591,6 +599,24 @@ components:
       required:
         - username
 
+      
+    UpdatePasswordByToken:
+      type: object
+      properties:
+        token:
+          description: token used for authentification
+          type: string
+        newPassword:
+          description: the new password to be updated
```

No need for articles when documenting parameters.
```suggestion
          description: new password to update
```

---

Fixed